release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -945,6 +945,13 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *openshift-apiserver*
 
+* Previously, a single failed request to the discovery endpoint of a service could make an Operator report `Available=False`. To increase resilience, a set of improvements have been introduced to prevent some Operators from reporting `Available=False` during an update due to various transient errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1948089[*BZ#1948089*])
+
+*OpenShift Update Service*
+
+* Previously, when creating an update service application through the web console, an invalid host error occurred. This occurred because of the default OpenShift Update Service (OSUS) application name was too long. A shorter default name is now in place and the error no longer occurs.
```

Comment:
```suggestion
* Previously, when creating an update service application through the web console, an invalid host error occurred. This occurred because the default OpenShift Update Service (OSUS) application name was too long. A shorter default name is now in place and the error no longer occurs.
```